lessons/Algebra2/logarithmic3-changing-scale/langs/en-us/index.adoc

```diff
@@ -55,34 +51,107 @@ Students plot a logarithmic sequence as points on a number line, and explore how
 
 === Launch
 
+Consider an exponential sequence, with a growth factor of 3: @hspace{1em} @big{@math{1, 3, 9, 27}...}
+
+@QandA{
+@Q{What are the next 2 numbers in the sequence?}
+@A{81, 243...}
+
+@Q{What does it mean that the @vocab{growth factor} of this sequence is 3?}
+@A{The values are doubling.}
+
+@Q{Why is the first number 1, when everything else is even?}
+@A{Because the number in the "zero position" is @math{3^0}, which is 1}
+
+@Q{What would this sequence look like, if graphed on the x-axis? The y-axis?}
+@A{Points bunched together near the origin, then rapidly spreading out as we move farther and father away}
+@A{x-axis: +
+@image{images/num-line-h.png, 400}}
+@A{y-axis: +
+@image{images/num-line-v.png, 50}}
+}
+
+@slidebreak
+
+@ifslide{
+[cols="^.^8,^.^1", frame="none", options="header"]
+|===
+| x-axis						| y-axis
+| @image{images/num-line-h.png} | @image{images/num-line-v.png}
+|===
+}
 
 === Investigate
 
 
+@teacher{Sync or pace students to *Slide 5: Changing Scale* of @starter-file{alg2-log-desmos}.}
+
+@lesson-instruction{
+- Return to the *Fitting Wealth-v-Health and Exploring Logarithmic Models Desmos file*.
+- You should now be on *Slide 5: Changing Scale*.
+- Use it to complete the first section of @printable-exercise{changing-scale-linear-logarithmic.adoc}.
+}
+
+@slidebreak
+
+When we changed the scale of the y-axis from `Linear` to `Logarithmic`, our intervals went from growing linearly to growing exponentially. This is like stretching and squishing our number line, so that each interval represents 10x the length of the interval before it.
+
+Our y-values are still growing exponentially! But now that the intervals are _also_ growing exponentially, this growth is cancelled out visually.
+
+@slidebreak
+
+@lesson-instruction{
+- Complete the second section of @printable-exercise{changing-scale-linear-logarithmic.adoc}.
+- Talk with your partner and/or other students nearby, to come up with an explanation of what is going on.
+- Then tackle the challenge questions below:
+}
+
+@QandA{
```

Comment:
I think this actually belongs at the end of @printable-exercise{changing-scale-linear-logarithimic.adoc}
But before we make that change...
I found it very confusing that the linear data appeared on the graph with the data points from the exponential sequence that had nothing to do with the line. I would suggest remove the folder with the linear function from slide 5 and add an additional slide to the Desmos file that is just for this experimentation.

Reply:
added to checklist